### PR TITLE
remove couch based XFormsSessions entirely

### PR DESCRIPTION
**Pull request by czue**
@gcapalbo this is https://github.com/dimagi/commcare-hq/pull/5787/ with one additional commit on top to actually remove the couch sessions https://github.com/dimagi/commcare-hq/commit/6a01756b70e7a2c859c5097cb86446194a885b09

Am guessing this one needs a bit of QA, though I did successfully run the `inbound_handlers` test.

### CODE REVIEW

File: corehq/apps/sms/handlers/keyword.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def sms_keyword_handler(v, text, msg):
     else:
         return handle_domain_keywords(v, text, msg, text_words, sessions)
 
-def _handle_structured_sms(domain, args, contact_id, session,
+def _handle_structured_sms(domain, args, contact_id, session_id,
```
Comment (dimagimon) [inline corehq/apps/sms/handlers/keyword.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/smsforms/app.py [hunk 2]
```diff
@@ -60,21 +61,27 @@ def start_session(domain, contact, app, module, form, case_id=None, yield_respon
...
-                            session_id = session_start_info.session_id,
-                            start_time=now, modified_time=now, 
-                            form_xmlns=form.xmlns,
-                            completed=False, domain=domain,
-                            app_id=app.get_id, user_id=contact.get_id,
-                            session_type=session_type)
+    session = SQLXFormsSession(
+        couch_id=uuid.uuid4().hex,  # for legacy reasons we just generate a couch_id for now
+        connection_id=connection_id,
+        session_id = session_start_info.session_id,
```
Comment (dimagimon) [inline corehq/apps/smsforms/app.py]: E251 unexpected spaces around keyword / parameter equals

**Comment by millerdev** [comment]
nice! this feels like it got significantly simpler

**Comment by snopoke** [comment]
We must remember to delete the couch views from prod. Also, should we purge all the docs from couch?

**Comment by czue** [comment]
i'd be in favor of purging docs after this is done. @gcapalbo do you see any issues with that?

**Comment by gcapalbo** [comment]
@czue No issues, I think that sounds right. Am hoping to get to this over the next couple days

**Comment by czue** [comment]
@gcapalbo any updates?

**Comment by gcapalbo** [comment]
@czue I haven't had time to dedicate to this yet, hopefully will get to it this week

**Comment by gcapalbo** [comment]
@czue This is good to go - I just added 2 commits with minor updates.  Do you know offhand if this will cause reindexing activity?  I figured no since we're removing all the design docs for this app, but wanted to make sure so we don't interfere with today's deploy.

**Comment by czue** [comment]
@gcapalbo thanks for updates. since we're deleting all views in the doc it shouldn't have any impact on the deploy

**Comment by czue** [comment]
:fireworks: :boom:

**Comment by snopoke** [comment]
:tada:

**Comment by czue** [comment]
deleted the design doc. :hammer: :hammer: :hammer:

**Comment by czue** [comment]
also made ticket to purge docs: http://manage.dimagi.com/default.asp?162295

**Pull request merged**